src/utils/fetchKeywords.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * A custom hook to fetch keywords from The Movie Database (TMDb) API based on a search query.
+ *
+ * @param {string} keyword - The search query for keywords.
+ * @returns {Promise<Array>} A Promise that resolves to an array of keyword objects.
+ */
+
+const options = {
```

Comment:
The options should be above the documentation since it's added for the fetch function.